From 315b354ac8010c9a20e638832b7998b777147904 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Mon, 26 May 2014 00:55:58 +1000
Subject: [PATCH 7/8] Change VideoFrame default value

dbb38743501ac4dd361b248fcaa337d00a85329c introduced a new behavior which is properly handled only in two places now, there are still other areas using VideoFrame.
Change default VideoFrame values so it uses the original behaviour for those instances
---
 mythtv/libs/libmythtv/frame.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/frame.h b/mythtv/libs/libmythtv/frame.h
index 3ed3a85..2a08d30 100644
--- a/mythtv/libs/libmythtv/frame.h
+++ b/mythtv/libs/libmythtv/frame.h
@@ -102,7 +102,7 @@ static inline void init(VideoFrame *vf, VideoFrameType _codec,
     vf->forcekey         = 0;
     vf->dummy            = 0;
     vf->pix_fmt          = 0;
-    vf->directrendering  = 0;
+    vf->directrendering  = 1;
 
     memset(vf->priv, 0, 4 * sizeof(unsigned char *));
 
-- 
1.7.10.2

